### ISSUE TITLE
When closing keyboard, content doesn't scroll back leaving ehite space in place of keyboard

### ISSUE THREAD

**Issue by Yuripetusko** [apache/cordova-plugin-wkwebview-engine#71]
Hi, this issue started with latest osx/xcode updates, nothing changed in code, and exactly same code and cordova/wkwebivew version worked fine a month ago. Now building the same code gives following issue, it is also described here:

https://forum.mendixcloud.com/link/questions/91916

Removing wkwebview fixes this issue

(I am not using any custom keyboard plugins)

Screenshots:
![img_0019](https://user-images.githubusercontent.com/666169/49395320-0ccb3380-f72e-11e8-9ab9-21db4f49fd90.jpg)

![img_0017](https://user-images.githubusercontent.com/666169/49395313-0a68d980-f72e-11e8-87fe-a828ea7cf14d.jpg)

![img_0018](https://user-images.githubusercontent.com/666169/49395317-0b9a0680-f72e-11e8-876a-036a60dc3069.jpg)

Here's a video of this happening:

https://drive.google.com/file/d/1PXkIu2Q5b0GDSrzKyrm_okyg2ciCVewK/view?usp=sharing

**Comment by Yuripetusko** [comment]
More info here: https://github.com/apache/cordova-ios/issues/417#issuecomment-432727776

**Comment by Yuripetusko** [comment]
For now solved this by downgrading to Xcode 9

**Comment by mwchambers** [comment]
For anyone still experiencing this problem the commit referenced above https://github.com/mwchambers/cordova-plugin-wkwebview-engine/commit/da67d6bb6ce8597c38fc69e66b84566e34efea8d 
 based on the changes here: https://github.com/ionic-team/cordova-plugin-ionic-webview/pull/201/commits seems to fix the problem.

**Comment by adam-h** [comment]
Works for us @mwchambers, thanks! It may be good to submit this as a PR.

**Comment by WuglyakBolgoink** [comment]
fyi: https://github.com/apache/cordova-plugin-wkwebview-engine/pull/75

**Comment by jacobweber** [comment]
The fix from @mwchambers worked for me as well. Could something like this be included in a release? I realize this is an Apple bug, but who knows if they'll ever fix it.

**Comment by shahdeep1989** [comment]
Why It's not fixed yet? Can any one provide me solution for this issue?

**Comment by be-ndee** [comment]
I have the same issue. I use the `cordova-plugin-keyboard` and can "solve" this issue with the following event listener. So everytime the keyboard will close, the window will scroll back to the correct position.

```typescript
window.addEventListener('keyboardWillHide', () => window.scrollTo(0, 0));
```

**Comment by c-emil** [comment]
See alternative solution without keyboard plugin: https://github.com/apache/cordova-ios/issues/417#issuecomment-523843485

**Comment by Lindsay-Needs-Sleep** [comment]
Since Apple should release the fix for this eventually, (they have already marked the bug as fixed...), you should be able to use cordova-plugin-wkkeyboardfix to fix the bug until Apple finally releases.

See plugin author's original comment:
https://github.com/apache/cordova-ios/issues/417#issuecomment-529032186

**Comment by dpogue** [comment]
This is resolved in iOS 13.4. For earlier versions of iOS, using the [cordova-plugin-wkkeyboardfix](https://www.npmjs.com/package/cordova-plugin-wkkeyboardfix) is probably the best solution. We will not be making any changes to Cordova to address this bug.
